debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresDefaultValueConverter.java

```diff
@@ -173,6 +176,8 @@ private static Map<String, DefaultValueMapper> createDefaultValueMappers(Timesta
         result.put("time", (c, v) -> timestampUtils.toLocalTime(extractDefault(v, "00:00")));
         result.put("timestamp", (c, v) -> timestampUtils.toOffsetDateTime(extractDefault(v, "1970-01-01")));
         result.put("timestamptz", (c, v) -> timestampUtils.toOffsetDateTime(extractDefault(v, "1970-01-01")).atZoneSameInstant(ZoneOffset.UTC));
+        result.put("_text", parseNullDefault((c, v) -> createStringArr(extractDefault(v, "{}"))));
```

Comment:
Is this issue for `_text/_varchar` only or are all array datatypes affected?

Reply:
Now that i think of it, looks like all array datatypes will be affected. Fixed only for these since this was impacting us. Will need a thorough review of other array datatypes. 

On the same note the default mapper did not have any array data types to begin with. These are the first few additions.

Reply:
@akanimesh7 Would it be possible to provide a sensible default - thus providing at least the empty List/Array for all array types and later on add support for the specific types?